addons/account/models/account_tax_report.py

```diff
@@ -75,11 +75,12 @@ def get_lines_in_hierarchy(self):
             lines_to_treat = list(to_yield.children_line_ids.sorted(lambda x: x.sequence)) + lines_to_treat[1:]
             yield to_yield
 
-    def get_checks_to_perform(self, d):
+    def get_checks_to_perform(self, amounts, carried_over):
         """ To override in localizations
         If value is a float, it will be formatted with format_value
         The line is not displayed if it is falsy (0, 0.0, False, ...)
-        :param d: the mapping dictionay between codes and values
+        :param amounts: the mapping dictionary between codes and values
+        :param carried_over: the mapping dictionary between codes and whether they are carried over
```

Comment:
I made it that way because it is the same way the information about the amounts was sent and I tried to keep a similar system.
This was only needed to exclude some lines from the verifications so it is, in my opinion, enough for this use case but if you think it should be changed I'm not against it.